### ISSUE TITLE
0.0.4-rc1 Uncaught promise rejection

### ISSUE THREAD

**Issue by akshat1** [akshat1/video-hoarder#21]
For some reason, the youtube-dl binary inside the docker image release-0.0.4-rc1 (obtained using curl) has issues executing.

If we `docker exec -it video-hoarder /bin/ash`, and then try to execute youtube-dl, we get the error `/usr/local/bin/youtube-dl: line 1: An: not found`. I have no idea what this means.

To debug, I created a temp container and followed the setup process step-by-step
```
docker run -it alpine /bin/ash
apk add --update nodejs npm curl ffmpeg python
curl -L https://yt-dl.org/downloads/latest/youtube-dl -o /usr/local/bin/youtube-dl
chmod a+rx /usr/local/bin/youtube-dl
5. youtube-dl --help
```

This worked. OTOH, it seems that when the same commands are executed in the Dockerfile, we end up with a defective youtube-dl binary which is only 58 bytes in size (while it should be 1.7M).

This corrupt youtube-dl binary causes `child_process.spawn` to fail inside the node app.

**Comment by akshat1** [comment]
This was fixed by switching from `RUN curl` to `ADD`.
See https://github.com/akshat1/video-hoarder/commit/079cb6f655cedcdc52678d58e600f37b70b0fb76#diff-3254677a7917c6c01f55212f86c57fbfR3

**Comment by akshat1** [comment]
Fixed with release-0.0.4.
